software/glasgow/applet/interface/uart/__init__.py

```diff
@@ -191,6 +191,8 @@ def build(self, target, args):
         bit_cyc,    self.__addr_bit_cyc    = target.registers.add_ro(32)
         rx_errors,  self.__addr_rx_errors  = target.registers.add_ro(16)
 
+        self.parity = args.parity
```

Comment:
This is an implementation detail; it should be done in a way like this so that inherited applets don't break:

```suggestion
        self.__has_parity = (args.parity != "none")
```